src/AppBundle/Command/DeleteUserCommand.php

```diff
@@ -0,0 +1,210 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace AppBundle\Command;
+
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\Question;
+use Doctrine\Common\Persistence\ObjectManager;
+
+/**
+ * A command console that deletes users and from the database.
+ * To use this command, open a terminal window, enter into your project
+ * directory and execute the following:
+ *
+ *     $ php app/console app:delete-user
+ *
+ * To output detailed information, increase the command verbosity:
+ *
+ *     $ php app/console app:delete-user -vv
+ *
+ * See http://symfony.com/doc/current/cookbook/console/console_command.html
+ *
+ * @author Oleg Voronkovich <oleg-voronkovich@yandex.ru>
+ */
+class DeleteUserCommand extends ContainerAwareCommand
```

Comment:
The official best practices don't mention the ContainerAware classes, so that means we don't discourage them.